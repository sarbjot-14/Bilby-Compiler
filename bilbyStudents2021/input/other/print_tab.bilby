main{
	
	print "tessting printing" $n; 
	
	print 3 $s$s 4 $ 5$7 $s$n;
}