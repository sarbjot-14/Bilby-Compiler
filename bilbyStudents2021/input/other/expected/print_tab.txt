tessting printing
3  457 
